commands/RunAll.java

```diff
@@ -1,20 +0,0 @@
-package org.usfirst.frc4904.standard.commands;
```

Comment:
while I agree that this file is no longer needed due to the changes in command groups, I think that its two children are still relevant and should be updated rather than deleted.

Reply:
How should this be fixed?